### ISSUE TITLE
Helm install rook operator, tolerationKey not used ?

### ISSUE THREAD

**Issue by PaulSka** [rook/rook#2955]
**Is this a bug report or feature request?**
* Bug Report

**Deviation from expected behavior:**
TolerationKey is not used ?

**Expected behavior:**

Agent and Discord on specific node with toleration

**How to reproduce it (minimal and precise):**
3 Nodes
Taint a node : kubectl taint nodes NODE1 is_storage=true:NoSchedule
Use a values.yaml :
```
agent:
  toleration: NoSchedule
  tolerationKey: is_storage
discover:
  toleration: NoSchedule
  tolerationKey: is_storage
```
Install rook chart : helm install ... rook-stable/rook-ceph -f values.yaml
Discover and Agent is on ALL nodes

**Environment**:
* OS (e.g. from /etc/os-release):
* Kernel (e.g. `uname -a`):
* Cloud provider or hardware configuration:
* Rook version (use `rook version` inside of a Rook Pod):
* Kubernetes version (use `kubectl version`):
* Kubernetes cluster type (e.g. Tectonic, GKE, OpenShift):
* Storage backend status (e.g. for Ceph use `ceph health` in the [Rook Ceph toolbox](https://rook.io/docs/Rook/master/toolbox.html)):

**Comment by PaulSka** [comment]
Daemont Set (Agent or Discover)

![image](https://user-images.githubusercontent.com/1415425/55867848-8dd68380-5b83-11e9-9e6b-3c17acff7087.png)

**Comment by PaulSka** [comment]
Hi,

I bump this issues.

**Comment by galexrt** [comment]
This Helm chart issue has been fixed in #1423, but as we are soon at the 1.0 release there will be no backport of this fix.

You can try to use a `master` helm chart "version" (`--version <version>`, see https://rook.io/docs/rook/v0.9/helm-operator.html) to workaround the issue till 1.0 has been released. When doing that make sure you have set the `image.tag` variable to, e.g., `v0.9.3` or the upcoming 1.0 release tag name.
